packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -45,6 +45,8 @@ interface RowsListProps {
   timeRange?: TimeRange;
   footerPaginationEnabled: boolean;
   initialRowIndex?: number;
+  // Boolean to specify if a data attribute containing row index is added to each row
+  showDataIndices: boolean;
```

Comment:
I don't understand this prop, it does not seem cause any visual change, yet named "showDataIndices". 

It just adds  data-row-index, how is that property used? why do we need it? If we need it can't we always render it?=

Reply:
We could always render it, or render it only if a `onCellClick` is provided.

Reply:
This prop is just to conditionally render the additional data attribute, which are only used when the table click callback function is defined, we didn't want to introduce unnecessary DOM attributes that could have performance impact on tables not using this feature (although the impact is negligible).

Reply:
If we want to render this for everyone that would simplify the code a bit, but I was trying to be overly cautious not to impact existing users to make it easier for this to get merged in so we can start building more interactivity with table ASAP.